provisioner/utils/setup-env-debian.sh

```diff
@@ -24,6 +24,8 @@ apt-get -y install rng-tools
 sed -i.bak 's@#HRNGDEVICE=/dev/null@HRNGDEVICE=/dev/urandom@' /etc/default/rng-tools
 service rng-tools start
 
+apt-get install -y locales
```

Comment:
How about we add a comment for the reason why we need to install this?

Reply:
The testing process would be broken due to "No such file or derictory: /etc/default/locale" in ubuntu16.04.

Reply:
Got it. So let's just add a comment to reveal your finding.

Reply:
Sure.